### PR TITLE
Fix serice owners use domain in query

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by dominikkowalski** [comment]
🆗

**Review by ar4s — APPROVED** [review]

**Review by szok — APPROVED** [review]

File: powerdns/models/powerdns.py [hunk 1]
```diff
@@ -278,7 +278,7 @@ def owners(self):
     @property
     def service_owners(self):
         """Returns owners by service unless direct_owners are set"""
-        owners = self.direct_owners.through.objects
+        owners = self.direct_owners.through.objects.filter(domain=self)
```
Comment (mkurek) [inline powerdns/models/powerdns.py:281]: i'd be enough to use `self. directowners_set` here :)

Comment (ghost) [reply powerdns/models/powerdns.py:281]: After simialr case last time, i was trying to did this but it didn't work, i think.
Isn't this accessible when `related_name` is added?

Comment (mkurek) [reply powerdns/models/powerdns.py:281]: `related_name` is the name from the other side of the relation. it should work, but i'll have to check in shell for exact name

**Pull request merged**